catalan.ini

```
@@ -0,0 +1,669 @@
+﻿[Header]
+Language = "Català"
+Author = "Minepika55, GuillemRoca"
+Author2 = "atemporalDarkness, Imsu"
+Version = "1.5.5"
+
+[Title]
+ControllerRecommended = "ES RECOMANA L'ÚS D'UN COMANDAMENT"
+BasedOnAGameBy = "Basat en un joc de"
+PressStart = "Prem Start"
+Options = "Opcions"
+Gallery = "Galeria"
+Credits = "Crèdits"
+Exit = "Sortir del Joc"
+Tip1 = "| - Seleccionar"
+Tip2 = "| - Tornar"
+GameSlot_NewGame = "Nova Partida"
+GameSlot_Easy = "Fácil"
+GameSlot_Normal = "Normal"
+GameSlot_Hard = "Difícil"
+GameSlot_Continue = "Continuar Partida"
+GameSlot_Copy = "Copiar Dades"
+GameSlot_Erase = "Esborrar Dades"
+GameSlot_EraseConfirm = "Borrar Dades Guardades?"
+GameSlot_OverwriteConfirm = "Sobrescriure Dades Guardades?"
+
+[GameOver]
+GameOver = "Fi de la partida"
+Continue = "Continuar"
+Exit = "Sortir"
+
+[GlobalOptions]
+Yes = "Si"
+No = "No"
+Enabled = "Activat"
+Disabled = "Desactivat"
+Exit = "Sortir"
+Exit_Tip = "Guardar els canvis i Sortir"
+ExitMain_Tip = "Guardar i tornar al Menú d'Opcions"
+Defaults = "Predeterminat"
+Defaults_Tip = "Restaurar configuració Predeterminada"
+
+[OptionsMain]
+Title = "Opcions"
+Display = "Opcions de visualització"
+Display_Tip = "Canviar opcions de Visualització"
+Sound = "Opcions de So"
+Sound_Tip = "Canviar el volum del So i la Música"
+Control = "Opcions de Control"
+Control_Tip = "Configuració de Control, teclat i comandament"
+
+[OptionsDisplay]
+Title_Display = "Mostrar Opcions"
+Fullscreen = "Pantalla Completa"
+Fullscreen_Tip = "Alternar entre Pantalla Completa i Finestra"
+DisplayScale = "Resolució"
+DisplayScale_Tip = "Defineix la mida de la Finestra del joc, conservant la relació d'aspecte"
+DisplayScale_Stretch = "Estira"
+WindowSize = "Tamany de la Finestra"
+WindowSize_Tip = "Defineix la mida de la Finestra del joc, conservant la relació d'aspecte"
+VSync = "Sincronització Vertical"
+VSync_Tip = "Sincronització vertical, elimina el trencament d'imatges. (Pot alentir el joc)"
+Title_HUD = "Opcions de la barra d'estat"
+HUDDisplay = "Mostra la barra d'estat"

```

Let´s make the text shorter so it doesn't conflict with option
```suggestion
HUDDisplay = "Mostrar barra d'estat"
```